autoload/lsp/omni.vim

```diff
@@ -279,12 +280,19 @@ function! s:sort_by_sorttext(i1, i2) abort
     endif
 endfunction
 
+" Create vim's completed items from LSP response.
+"
 " options = {
 "   server: {}, " needs to be server_info and not server_name
 "   position: lsp#get_position(),
 "   response: {}, " needs to be the entire lsp response. errors need to be
 "   handled before calling the fuction
 " }
+"
+" * The returned` startcol` may be the same as the cursor position, in which case you need to decide which one to use.
+"
+" @return { 'items': v:completed_item[], 'incomplete': v:t_bool, 'startcol': number }
```

Comment:
I've added the `startcol` for return value. (It can be used by asyncomplete-lsp.vim, I think)